sr/robot3/robot.py

```diff
@@ -370,39 +373,36 @@ def wait_start(self) -> None:
         """
         LOGGER.info("Waiting for start signal")
 
-        start_event = asyncio.Event()
+        start_event = threading.Event()
 
         astoria_start = WaitForStartButtonBroadcastConsumer(
-            self._verbose,
-            None,  # Don't pass a config
+            self._mqtt,
             start_event,
         )
 
-        async def wait_for_physical_start() -> None:
-            self.power_board.piezo.buzz(timedelta(seconds=0.1), Note.A6)
-            counter = 0
-            led_state = False
-            _ = self.power_board.start_button.is_pressed
-            while not self.power_board.start_button.is_pressed and not start_event.is_set():  # noqa: E501
-                if counter % 6 == 0:
-                    led_state = not led_state
-                    self.power_board._run_led.state = led_state
-                    self.kch.start = led_state
-                await asyncio.sleep(0.05)
-                counter += 1
-            start_event.set()
-            # Turn on the LED now that we are starting
-            self.power_board._run_led.state = True
-
-            # Turn off the KCH Start LED.
-            self.kch.start = False
-
-        loop.run_until_complete(
-            asyncio.gather(
-                astoria_start.run(),
-                wait_for_physical_start(),
-            ),
-        )
+        self.power_board.piezo.buzz(timedelta(seconds=0.1), Note.A6)
+        counter = 0
+        led_state = False
+        _ = self.power_board.start_button.is_pressed
+        while (
+            not self.power_board.start_button.is_pressed
+            and not start_event.is_set()
+        ):
```

Comment:
```suggestion
        while not self.power_board.start_button.is_pressed and \
            not start_event.is_set():
```

A continuation line is much better here.